packages/client/manifest.json

```diff
@@ -4435,6 +4435,48 @@
                 "key": "row"
               }
             ]
+          },
+          {
+            "label": "Side panel fields",
+            "type": "fieldConfiguration",
+            "key": "sidePanelFields",
+            "nested": true,
+            "dependsOn": {
+              "setting": "clickBehaviour",
+              "value": "details"
+            }
+          },
+          {
+            "label": "Show side panel delete button",
+            "type": "boolean",
+            "key": "sidePanelShowDelete",
+            "nested": true,
+            "dependsOn": {
+              "setting": "clickBehaviour",
+              "value": "details"
+            }
+          },
+          {
+            "label": "Side panel save label",
+            "type": "text",
+            "key": "sidePanelSaveLabel",
+            "defaultValue": "Save",
+            "nested": true,
+            "dependsOn": {
+              "setting": "clickBehaviour",
+              "value": "details"
+            }
+          },
+          {
+            "label": "Side panel delete label",
+            "type": "text",
+            "key": "sidePanelDeleteLabel",
+            "defaultValue": "Delete",
+            "nested": true,
+            "dependsOn": {
+              "setting": "sidePanelShowDelete",
```

Comment:
NAB: We actually should update this to depend on clickBehaviour = details as well. We technically want it to depend on 2 conditions (both details behaviour, and also sidePanelShowDelete) but we can't do both. Choosing the behaviour setting prevents this setting being shown when the behaviour is "run actions" which is the better choice in my opinion. 

Having a setting for the delete button text always present (even if you choose to hide the button) isn't as strange as seeing the setting when you aren't even using a side panel.

Reply:
Ah hadn't considered this, good catch, sucks there can't be two conditions.